### DISCUSSION TITLE
Out of stock everywhere and removed from Pi Supply

### DISCUSSION THREAD

**Discussion by Blockguy24** [PiSupply/PiJuice#1093] [Q&A] [answered]
I was just looking into UPS solutions for my RPi and came across PiJuice. It looked very promising until I saw that it appears to be out of stock everywhere. I tried looking for it on the Pi Supply store, but it appears to not exist anymore (https://uk.pi-supply.com/products/pijuice-standard is a 404). I'm just wondering if the product has been discontinued? And if so, what are some good alternative products?

**Comment by tvoverbeek** [comment]
The PiJuice HAT is shown as out-of-stock.
However the PiJuice Zero is still available

**Reply by Blockguy24** [reply]
I am aware, but is the Zero variant compatible with regular RPi's or only for RPi Zero?

**Comment by shawaj** [comment, marked as answer]
We have 2k pcs being manufactured as we speak. 

We believe they will be finished by mid September so you should see stock by end of September in various places. 

The Pi Supply website automatically removes products that are out of stock which is why you don't see it.

**Reply by Blockguy24** [reply]
Ohhhh that makes sense, I don't think I've seen any other website do that so that's why I got confused. Thank you very much!

**Reply by shawaj** [reply]
Actually it was a misconfiguration on our website. Sorry for the confusion!

https://uk.pi-supply.com/collections/pijuice/products/pijuice-portable-power-raspberry-pi

**Reply by 1anonyy** [reply]
> Actually it was a misconfiguration on our website. Sorry for the confusion!
> 
> https://uk.pi-supply.com/collections/pijuice/products/pijuice-portable-power-raspberry-pi

@shawaj  Hi Aaron, Are you still making this product?

**Comment by lotte67890** [comment]
May I kindly ask when the pijuice will be available again? I still can not find any store having it in stock...
Thank you
Lotte

**Reply by lotte67890** [reply]
Question answered, meanwhile it seems to be in stock, but unfortunately there´s no payment available at the checkout site...

**Reply by shawaj** [reply]
It will be in stock in many places next week. Including KJD Electronics, KUBII, Digi-Key, Distrelec and a few others

**Comment by ArjenR49** [comment]
For an alternative solution have a look at UUGear's WittyPi HATs. I'm using a WittyPi L3v7 with an 18650 Li cell on a Pi4B. In my case it's just for making sure the OS drive isn't damaged by a blackout, though. The WittyPi L3v7 is also a lot cheaper.

**Reply by bunjg** [reply]
What's the latest on inventory?  The website is down again and the only place they are available is Amazon for 2x the price shown on digikey.  I'd like to buy 3 more or I'm going to have to redesign/update software to go with a different UPS.

**Comment by tvoverbeek** [comment]
Pi Supply has been dissolved.
See https://find-and-update.company-information.service.gov.uk/company/08518826
So either find existing stock somewhere or redesing with a different UPS.

**Comment by lotte67890** [comment]
Sad to see that you stopped making this fabulous product! I use two of them and never encountered any problems!
All the best for your future.

L

**Comment by Canardwcbang** [comment]
Hi,
Sad to see it stopped also. I was testing a few on raspberrypi 4b and was aiming at buying more.
If someone has some stock or alternatives, I am all ears !

Marc
